src/main/java/com/cburch/logisim/fpga/hdlgenerator/Hdl.java

```diff
@@ -99,6 +213,13 @@ public static String vectorLoopId() {
     return isVhdl() ? " DOWNTO " : ":";
   }
 
+  public static String splitVector(int start, int end) {
+    if (start == end) return LineBuffer.format("{{1}}{{2}}{{3}}", bracketOpen(), start, bracketClose());
```

Comment:
What about using `formatHdl()` here instead?

Reply:
There were a lot of changes to the LineBuffer and the HDL generation over the last weeks, I couldn't quite hold up.
When and how is this function used?

I could use the placeholders here. (`LineBuffer.format("{{<}}{{1}}{{>}}", start)`)

Reply:
Changes to LineBuffer are backward compatible in terms of API. And no, you cannot HDL placeholders with plain `format()` as HDL pairs are not defined in that case and there's no way for you to set it up. That's why you got `formatHdl()` is exactly like `format()` but it operates in "HDL context". See: https://github.com/logisim-evolution/logisim-evolution/blob/develop/src/main/java/com/cburch/logisim/util/LineBuffer.java#L616